src/components/icons/md/Ai.svg

```diff
@@ -0,0 +1,6 @@
+<svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
+  <path
+    d="M15.417 11.0741C15.513 11.0741 15.574 11.0131 15.595 10.9171C15.786 9.80308 15.779 9.78908 16.955 9.55708C17.051 9.53708 17.112 9.48108 17.112 9.37908C17.112 9.28308 17.051 9.22208 16.955 9.20808C15.779 8.98908 15.8 8.96808 15.595 7.84808C15.575 7.75208 15.513 7.68408 15.417 7.68408C15.321 7.68408 15.26 7.75208 15.239 7.84808C15.034 8.95508 15.062 8.96808 13.879 9.20808C13.783 9.22208 13.722 9.28308 13.722 9.37808C13.722 9.48108 13.783 9.53608 13.886 9.55708C15.055 9.78208 15.048 9.78908 15.239 10.9171C15.259 11.0131 15.321 11.0741 15.417 11.0741ZM12.095 15.9211C12.245 15.9211 12.361 15.8111 12.382 15.6681C12.614 13.8701 12.662 13.8561 14.549 13.5081C14.6176 13.5006 14.6809 13.4677 14.7264 13.4159C14.772 13.3641 14.7964 13.2971 14.795 13.2281C14.795 13.0781 14.685 12.9611 14.542 12.9481C12.669 12.6881 12.614 12.6331 12.382 10.7941C12.362 10.6441 12.252 10.5341 12.095 10.5341C11.945 10.5341 11.835 10.6441 11.815 10.8011C11.595 12.5991 11.521 12.5991 9.64702 12.9471C9.49702 12.9671 9.39502 13.0771 9.39502 13.2271C9.39502 13.3851 9.49702 13.4871 9.67502 13.5071C11.521 13.7951 11.595 13.8571 11.815 15.6541C11.835 15.8121 11.945 15.9211 12.095 15.9211ZM16.915 23.4611C17.126 23.4611 17.29 23.3111 17.325 23.0851C17.823 19.4151 18.335 18.8331 21.98 18.4231C22.0848 18.4165 22.1831 18.3706 22.2555 18.2946C22.3278 18.2186 22.3687 18.118 22.37 18.0131C22.37 17.7931 22.205 17.6301 21.98 17.5961C18.37 17.1651 17.857 16.6391 17.324 12.9341C17.284 12.7141 17.126 12.5641 16.914 12.5641C16.702 12.5641 16.538 12.7141 16.504 12.9341C16.004 16.6111 15.492 17.1921 11.849 17.5961C11.623 17.6231 11.459 17.7941 11.459 18.0131C11.459 18.2251 11.623 18.3961 11.849 18.4231C15.451 18.9151 15.95 19.3871 16.504 23.0851C16.544 23.3111 16.703 23.4611 16.915 23.4611Z"
+    fill="#172C4D" />
+</svg>
+  
```

Comment:
```suggestion
  
  
```

Reply:
@luk-zap I got confused because I saw this icon 
![Screenshot 2023-11-07 at 13 26 11](https://github.com/archilogic-com/honeycomb/assets/22019313/4ad548d5-980a-47aa-b70c-12a0770a738b)
github bug?